### PR TITLE
Fix: Fixed footer sections vertical alignment issue

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
Footer above/below and primary header vertical alignment was not working due to some Divider CSS. I have added a fix for the Alignment issue.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/footer/below-footer/assets/js/customizer-preview.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
     astra_css(
         'astra-settings[hbb-footer-vertical-alignment]',
         'align-items',
-        selector + ' .ast-builder-grid-row'
+        selector + ' .ast-builder-grid-row, ' + selector + ' .ast-builder-grid-row .site-footer-section'
```
Comment (premanshup) [inline inc/builder/type/footer/below-footer/assets/js/customizer-preview.js:23]: @sushmak02 Do we really need  `.ast-builder-grid-row .site-footer-section`
Or can it managed by just once class -  `.site-footer-section`

File: inc/builder/type/footer/above-footer/dynamic-css/dynamic.css.php [hunk 2]
```diff
@@ -50,9 +50,11 @@ function astra_fb_above_footer_dynamic_css( $dynamic_css, $dynamic_css_filtered
 
 		$selector                            => astra_get_responsive_background_obj( $footer_bg, 'desktop' ),
 		$selector . ' .ast-builder-grid-row' => array(
-			'align-items'     => astra_get_option( 'hba-footer-vertical-alignment' ),
 			'grid-column-gap' => astra_get_css_value( $inner_spacing_desktop, 'px' ),
 		),
+		$selector . ' .ast-builder-grid-row, ' . $selector . ' .ast-builder-grid-row .site-footer-section' => array(
```
Comment (premanshup) [inline inc/builder/type/footer/above-footer/dynamic-css/dynamic.css.php:55]: @sushmak02 Do we really need `.ast-builder-grid-row .site-footer-section`
Or can it managed by just once class - `.site-footer-section`

File: inc/builder/type/footer/primary-footer/dynamic-css/dynamic.css.php [hunk 3]
```diff
@@ -63,9 +63,11 @@ function astra_fb_primary_footer_dynamic_css( $dynamic_css, $dynamic_css_filtere
 
 		$selector                            => astra_get_responsive_background_obj( $footer_bg, 'desktop' ),
 		$selector . ' .ast-builder-grid-row' => array(
-			'align-items'     => astra_get_option( 'hb-footer-vertical-alignment' ),
 			'grid-column-gap' => astra_get_css_value( $inner_spacing_desktop, 'px' ),
 		),
+		$selector . ' .ast-builder-grid-row, ' . $selector . ' .ast-builder-grid-row .site-footer-section' => array(
```
Comment (premanshup) [inline inc/builder/type/footer/primary-footer/dynamic-css/dynamic.css.php:68]: @sushmak02 Do we really need `.ast-builder-grid-row .site-footer-section`
Or can it managed by just once class - `.site-footer-section`

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/builder/type/footer/below-footer/assets/js/customizer-preview.js (hunk 1 above)
Comment (sushmak02) [reply inc/builder/type/footer/below-footer/assets/js/customizer-preview.js]: Done!

File: inc/builder/type/footer/above-footer/dynamic-css/dynamic.css.php (hunk 2 above)
Comment (sushmak02) [reply inc/builder/type/footer/above-footer/dynamic-css/dynamic.css.php]: Done!

File: inc/builder/type/footer/primary-footer/dynamic-css/dynamic.css.php (hunk 3 above)
Comment (sushmak02) [reply inc/builder/type/footer/primary-footer/dynamic-css/dynamic.css.php]: Done!

**Review by premanshup — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**